src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card_img"
+        src="./images/imac.jpeg"
+        alt="photo"
+      >
+
+      <p class="card__title title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+      <p class="card__code code">Product code: 195434</p>
+
+      <div class="card__rating rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price price">
+        <p class="price-text">Price:</p>
+        <p class="price-money">$2,199</p>
+      </div>
+
+      <a
+        class="card__link link"
+        href="#"
+        data-qa="hover"
+      >
+        BUY
```

Comment:
```suggestion
        Buy
```
do uppercase via CSS